vis.c

```diff
@@ -286,17 +286,18 @@ static void window_draw_colorcolumn(Win *win) {
 		if (l->lineno != lineno) {
 			line_cols = 0;
 			line_cc_set = false;
-			lineno = l->lineno;
+			if (!(lineno = l->lineno))
+				break;
```

Comment:
why not the much simpler and clearer:
```
lineno = l->lineno;
if (! lineno)
```

Reply:
I searched through the codebase and I saw similar checks, so that's why I chose this style.